docs/usage.md

```diff
@@ -90,7 +90,7 @@ configManagementPlugins: |
       args: ["helm dependency build"]
     generate:
       command: ["sh", "-c"]
-      args: ["helm template $ARGOCD_APP_NAME ${helm_args} . --include-crds | argocd-vault-plugin generate -"]
+      args: ["helm template $ARGOCD_APP_NAME ${helm_args} . --include-crds --kube-version $KUBE_VERSION --api-versions $KUBE_API_VERSIONS | argocd-vault-plugin generate -"]
```

Comment:
I think instead of continuing to add variables here to this example, maybe we can link to the helm docs? That we we don't have to call out all args since its just a helm template command and can rely on the helm docs.  We can  suggest using these particular options for the reason described in the PR.

Reply:
I disagree here. The argo-vault-plugin documentation should have some kind of 'works out of the box' experience. No one will go through the helm docs and the helm docs does not describe the problem. The options and parameters are manadory, otherwise the code from documentation will lead into unexpected behaivors. Thats the point, why the options should be added there.

From my point of view, the general issue is that argo-vault-plugin can not re-use the existing helm logic from ArgoCD. But the dicussion feels off-topic here.

Reply:
@jkroepke can you also add to the manifests then? That would be the place for 'works out of the box' experience.

Reply:
> From my point of view, the general issue is that argo-vault-plugin can not re-use the existing helm logic from ArgoCD. But the dicussion feels off-topic here.

That is a larger Argo CD issue that we cant solve